apps/gpc-prover-client/package.json

```diff
@@ -10,10 +10,9 @@
     "localenv": "cp .env.example .env.local"
   },
   "dependencies": {
-    "@pcd/gpc": "^0.1.0",
-    "@pcd/pod": "^0.2.0",
-    "@semaphore-protocol/identity": "^3.15.2",
-    "json-bigint": "^1.0.0",
+    "@pcd/gpc": "^0.3.0",
+    "@pcd/pod": "^0.4.0",
+    "semaphore-identity-v4": "npm:@semaphore-protocol/identity@^4.0.3",
```

Comment:
Rob ended pointing out a problem with this renaming which can be troublesome when built together with  other packages which don't rename it.  I suggest just using semaphore-protoco/identity directly, since there's no need to refer to V3 anymore.  We don't want to encourage devs to copy this renaming pattern if we've determined it's problematic.

Reply:
I saw Rob's PR here: https://github.com/proofcarryingdata/zupass/pull/2019.
I encountered the exact problem he mentioned there. That is why I renamed this too.

Is his change released?
I'm going to bump @pcd/gpc again after his change gets released, so I won't rename this anymore.

Reply:
The wrapper isn't released to NPM yet.  Rob's also still seen some confusing behavior with different versions of Semaphore Identity, as have I.  I'm not sure we know the right solution here.  It's most important that ZuKYC have something which works, and secondarily that it be a good example for others to follow.
This renaming might be safer in making sure you get the version you asked for, but might lead to getting a different version than the GPC libraries get.  I've got some fixes to make the check for that in proving less restrictive in this PR: https://github.com/proofcarryingdata/zupass/pull/2056